scripts/managedb/fix_location_gjson.py

```diff
@@ -58,18 +58,40 @@ def old_coordinates_format(doc):
 DB = sys.argv[1]
 COL = 'entities'
 
+# Warn user
+print "WARNING!!!! This script modifies your '%s' database. It is STRONGLY RECOMMENDED that you" % DB
```

Comment:
The part after the final '%' are variables used to "fill the gaps" in the string. Basically, the same that printf does in C (probably this Pyton-ic thing was copyed from C :))